lib/matplotlib/tests/test_axes.py

```diff
@@ -482,6 +482,9 @@ def test_hexbin_extent():
 
     ax.hexbin(x, y, extent=[.1, .3, .6, .7])
 
+def test_hexbin_empty():
+    # From #3886: creating hexbin from empty dataset raises ValueError
+    ax.hexbin([], [])
```

Comment:
You need to create the Axes first.  Also, I think you will want to use the @cleanup decorator.